E_01_01/E_01_01/E_01_01.cpp

```diff
@@ -0,0 +1,7 @@
+using namespace std;
+int main()
+{
+	cout << "���߂Ă�C++�v���O�����B\n";
+	cout << "��ʂɏo�͂��Ă��܂��B\n";
```

Comment:
https://help.github.com/ja/github/setting-up-and-managing-your-github-user-account/inviting-collaborators-to-a-personal-repository